pkgs/by-name/op/open-webui/package.nix

```diff
@@ -207,7 +208,10 @@ python312.pkgs.buildPythonApplication rec {
     changelog = "https://github.com/open-webui/open-webui/blob/${src.tag}/CHANGELOG.md";
     description = "Comprehensive suite for LLMs with a user-friendly WebUI";
     homepage = "https://github.com/open-webui/open-webui";
-    license = lib.licenses.mit;
+    license = [
+      lib.licenses.mit
+      lib.licenses.open-webui
+    ];
```

Comment:
I see no signs of MIT here, before or after. The old contributions to the project are `BSD-3-Clause`, but I’m not sure it makes sense to record that given that the requirements of the new licence are a strict superset.

Reply:
That's indeed a mistake, going to fix it now.